### ISSUE TITLE
Reload regression introduced in "Improved autoreload (#1555)"

### ISSUE THREAD

**Issue by fundamental** [fonsp/Pluto.jl#1582]
First off, thanks for all of the work into Pluto.jl. It's pretty darn rad and it an enjoyable tool to use while exploring data and building algorithms.

As per the issue, the referenced pull request changes some code in SessionActions.jl to: (pruning commented out code)

```
            modified_time = mtime(nb.path)
            if modified_time - nb.last_save_time < session.options.server.auto_reload_from_file_cooldown
                # @info "Modified time is very close to my last save time, not reloading from file."
            else
                update_from_file_throttled()
            end
```

This reads perfectly fine. The file has changed, so grab the modification time, then see if it merits a reload. Unfortunately it doesn't work with my editor (vim). When editing the file, changes occur, this block of code is run, but `mtime(nb.path)` returns a value of zero. My guess without further investigation is that vim removes the file and then writes the new file which provides a very narrow window for julia to perform a `stat()` operation on a file that doesn't exist resulting in `mtime()` being the unix epoch. The old approach of using `time()` appears to resolve the issue with reloading not working on my system.

Put simply, `mtime(nb.path)` results in no reloads occurring, while `time()` results in the old working reload behavior.

**Comment by fonsp** [comment]
Can you try out https://github.com/fonsp/Pluto.jl/pull/1584 ?
